### PR TITLE
Feature/visits

### PR DESCRIPTION
**Pull request by HFG43**
Updated query:
- [ ] Details for most recent visit: animal information, vet information, and date of visit.
Screenshot from 2023-08-18 22-39-20

Hi there! I'm sending this pull request for you to review my code. Thanks in advance for your feedback!
for this Milestone, I have done the following:

- Use gitflow
- Add Join Table for visits
- Make queries using many-2-many relationships

Please find below the screenshots of my queries results:

- [ ] Who was the last animal seen by William Tatcher?
Screenshot from 2023-08-18 16-57-03

- [ ] How many different animals did Stephanie Mendez see?
Screenshot from 2023-08-18 17-30-43

- [ ] List all vets and their specialties, including vets with no specialties.
Screenshot from 2023-08-18 17-45-11

- [ ] List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
Screenshot from 2023-08-18 17-51-18

- [ ] What animal has the most visits to vets?
Screenshot from 2023-08-18 18-05-26

- [ ] Who was Maisy Smith's first visit?
Screenshot from 2023-08-18 18-12-43

- [ ] Details for most recent visit: animal information, vet information, and date of visit.
Screenshot from 2023-08-18 18-49-40

- [ ] How many visits were with a vet that did not specialize in that animal's species?
Screenshot from 2023-08-18 19-22-15

- [ ] What specialty should Maisy Smith consider getting? Look for the species she gets the most.
Screenshot from 2023-08-18 21-50-41

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -55,3 +55,21 @@ SELECT animals.name, owners.full_name, species.name FROM animals INNER JOIN owne
...
+SELECT vets.name, specializations.vets_id, species.name FROM specializations FULL OUTER JOIN vets ON vets_id = vets.id FULL OUTER JOIN species ON species_id = species.id;
+
+SELECT animals.name, vets.name, day_of_visit FROM visits JOIN animals ON animals_id = animals.id JOIN vets ON vet_id = vets.id WHERE vets.name LIKE '%Mendez' AND day_of_visit >= '2020-04-01' AND day_of_visit <= '2020-08-30';
+
+SELECT animals.name, COUNT(day_of_visit) FROM visits JOIN animals ON animals_id = animals.id GROUP BY animals.name ORDER BY COUNT(day_of_visit) DESC LIMIT 1;
+
+SELECT day_of_visit, animals.name, vets.name FROM visits JOIN vets ON vet_id = vets.id JOIN animals ON animals_id = animals.id WHERE vets.name LIKE '%mith%' ORDER BY day_of_visit LIMIT 1;
+
+SELECT animals.name as animal_name, vets.name as vet_name, day_of_visit FROM visits JOIN animals ON animals_id = animals.id JOIN vets ON vet_id = vets.id ORDER BY date_of_graduation DESC LIMIT 1;
+
```
Comment (abdulhamiid) [inline queries.sql:72]: image

Please add the **correct** query and the result screenshots in your pull request description. 🛠🤗

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @HFG43 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Understand the different types of relationships between tables.
- ✔️ Prepare complex queries that answer analytical questions.
- ✔️ Use primary key & foreign key mechanisms for joining tables.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (HFG43) [reply queries.sql:72]: @abdulhamiid Thanks!!

**Review by usorfaitheloho — APPROVED** [review]
Hi @HFG43  , 
# STATUS: APPROVED! 🥇 💯 

done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**